proto/eni.proto

```diff
@@ -23,7 +23,7 @@ message Eni {
     string vnet = 6;
     // Optional
     // Private Link encoding for IPv6 SIP transpositions
-    optional types.IpPrefix pl_sip_encoding = 7 [deprecated = true];
```

Comment:
This is still marked as deprecated in the HLD, is the field needed again?

Reply:
yes, this is no longer deprecated.